packages/wallet/ui/src/contexts/Application.jsx

```diff
@@ -9,8 +11,38 @@ import {
   useReducer,
 } from 'react';
 
+// TODO: Graduate to show mainnet in production.
+const SHOW_MAINNET = process.env.NODE_ENV === 'development';
+export const DEFAULT_WALLET_CONNECTIONS = [
+  ...(window && window.location.hostname !== 'localhost'
+    ? [{ label: window.location.hostname, url: window.location.origin }]
+    : [{ label: 'Local Solo', url: 'http://localhost:8000' }]),
+  ...(process.env.NODE_ENV === 'development'
+    ? [
+        {
+          label: 'Agoric Stage',
+          url: 'https://stage.agoric.net/network-config',
+        },
+      ]
+    : []),
+  ...(SHOW_MAINNET
+    ? [
+        {
+          label: 'Agoric Mainnet',
+          url: 'https://main.agoric.net/network-config',
+        },
+        {
+          label: 'Agoric Devnet',
+          url: 'https://devnet.agoric.net/network-config',
+        },
+      ]
+    : []),
+];
+
 export const ApplicationContext = createContext();
 
+export const DEFAULT_WALLET_CONNECTION = DEFAULT_WALLET_CONNECTIONS[0];
```

Comment:
these two very similar names seem like a smell. also is this export used anywhere?